src/logius/seo.js

```diff
@@ -0,0 +1,201 @@
+// @ts-check
+// Module w3c/seo
```

Comment:
This module is still using a lot of W3C stuff...

Reply:
Switched logius profile to the w3c version.